Utilities/Dox/PythonScripts/FileManDataToHtml.py

```diff
@@ -714,52 +832,48 @@ def findSubValue(self,dataEntry,search,multval, id):
   def _generateDataTableHtml(self, fileManData, fileNo, outDir):
     isLargeFile = len(fileManData.dataEntries) > 4500
     tName = normalizePackageName(fileManData.name)
+    print "Writing info for %s" % fileNo
```

Comment:
Should this be logged instead of printed?

Reply:
Most likely.  I'll remove it.